src/main/java/com/esri/samples/localserver/local_server_dynamic_workspace_shp/README.md

```diff
@@ -0,0 +1,42 @@
+<h1>Local Server Dynamic Workspace Shapefile</h1>
+
+<p>Demonstrates how to dynamically add a local shapefile to a map using the Local Server. </p>
+
+<p><b>Note:</b> Local Server is not supported on MacOS</p>
+
+<img src="LocalServerDynamicWorkspaceShapefile.png"/>
+
+<h2>How to use the sample</h2>
+
+<p>A Local Server and Local Map Service will automatically be started and once running a Feature Layer will be created and added to the map. </p>
```

Comment:
Would change this to read like below,
A Local Server will automatically start once application is running. Select a shapefile using the `Choose Shapefile` button which will automatically start a Local Map Service where a Dynamic Shapefile Workspace will be set and displayed to the Map View.